jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/template/jsMessages.jsp

```diff
@@ -183,6 +183,6 @@ $labelNotLoggedInError = '<spring:message code="label.not.logged.in.error" htmlE
 $pollItemsSize = '<spring:message code="poll.items.size" htmlEscape="true" text=""/>';
 $pollItemLength = '<spring:message code="poll.item.length" htmlEscape="true" text=""/>';
 
-$labelConnectionLostGenericError = '<spring:message code="label.connection.lost.genericError" htmlEscape="true"/>';
+$labelConnectionLostGenericError = "<spring:message code="label.connection.lost.genericError" htmlEscape="false"/>";
```

Comment:
This would allow for HTML Injections, wouldn't it? If we change the message at some point in the future there is a risk that we break HTML.